### DISCUSSION TITLE
„Cannot open the file as archive error code: 2“ what to do?

### DISCUSSION THREAD

**Discussion by Quallenknoedel** [aonez/Keka#1122] [Q&A] [answered]
Hey,
Keka says „cannot open the file as archive // error code 2“ when I tried to unpack a movie. I renamed all four parts of the movie into "part1" "part2" etc. and hope, it didn't make any thing worse. Any help will be appreciated!

========
OS: Version 12.4 (Build 21F79) (arm64)
Keka: v1.2.54-r4983 (WEB) (Sandboxed) (de)
Format detected: RAR Legacy
Binary used: kekaunrar
Arguments: (
    x,
    "-idcd",
    "-kb",
    "-x*/__MACOSX",
    "-y",
    "/Users/(my user name)/Downloads/Old/part1.rar",
    "/Users/(my user name)/Downloads/Old/part1.kextraction/Operation/"
)

/Users/(my user name)/Downloads/Old/part1.rar is not RAR archive
No files to extract

OS: Version 12.4 (Build 21F79) (arm64)
Keka: v1.2.54-r4983 (WEB) (Sandboxed) (de)
Format detected: RAR Legacy
Binary used: keka7zz
Arguments: (
    x,
    "/Users/(my user name)/Downloads/Old/part1.rar",
    "-o/Users/(my user name)/Downloads/Old/part1.kextraction/Operation",
    "-snld",
    "-aou",
    "-xr!__MACOSX",
    "-bsp1"
)

7-Zip (z) 21.07 (arm64) : Copyright (c) 1999-2021 Igor Pavlov : 2021-12-26 : Modified by aone for Keka
 64-bit arm_v:8 locale=en_US.UTF-8 Threads:8, ASM

Scanning the drive for archives:
  0M Scan /Users/(my user name)/Downloads/Old/
1 file, 2097152000 bytes (2000 MiB)

Extracting archive: /Users/(my user name)/Downloads/Old/part1.rar
  0% Open  0%
Can't open as archive: 1
Files: 0
Size:       0
Compressed: 0

ERROR: /Users/(my user name)/Downloads/Old/part1.rar
Cannot open the file as archive

Error code 2

**Comment by aonez** [comment, marked as answer]
I've replied you on the issue you've created: https://github.com/aonez/Keka/issues/1123
